storage/interceptors/notifications_interceptor.go

```diff
@@ -96,6 +102,21 @@ func (ni *NotificationsInterceptor) OnTxUpdate(h storage.InterceptUpdateOnTxFunc
 			return nil, err
 		}
 
+		// If the updated object has just become ready, then create a CREATED notification for it
+		if !oldObject.GetReady() && updatedObject.GetReady() {
```

Comment:
It seems like you've forgotten to strip the updatedObject's labels like it's done a few lines below.

Reply:
This is because, when we create an object we don't have label changes, thus we need the created object with all its labels so it can be correctly applied by the agent